.changeset/yellow-goats-travel.md

```diff
@@ -2,4 +2,95 @@
 '@openfn/language-fhir': major
 ---
 
-`fhir` redesign
+### Added
+
+- Introduced a new `RequestOptions` for the HTTP request:
```

Comment:
RequestOptions and GetOptions arepurely documentation. We don't need to mention them at all in the changelog.

If you think it's really important for users to know you could add a general line like "Improved type definitions for request and get"

But there's no call to duplicate all the documentation here.

Reply:
Should i keep the function signature docs ?

Reply:
Yep, we can keep those